src/main/java/fr/foodlimit/api/food/FoodController.java

```diff
@@ -28,8 +46,14 @@ public FoodController() {
    * @return
    */
   @GetMapping
-  public List<Food> getFoods(@PathVariable("placeId") Long placeId) {
-    return foodService.getFoods(placeId);
+  public ResponseEntity<List<Food>> getFoods(HttpServletRequest request, @PathVariable("placeId") Long placeId) {
+    Place place = placeService.getPlace(placeId);
+
+    if (!this.environment.getActiveProfiles()[0].equals("test")) {
+      if (!checkPlace(request, place)) return ResponseEntity.status(HttpStatus.FORBIDDEN).build();
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Merge this if statement with the enclosing one. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1066)